config/routes.rb

```diff
@@ -5,6 +5,9 @@
   post 'login' => 'sessions#create'
   get 'logout' => 'sessions#destroy', as: :logout
 
+  get 'login/sc' => 'sessions#new_sc', as: :sc_login
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.